com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/RenderPass/CustomPass/CustomPassCommon.hlsl

```diff
@@ -44,6 +44,29 @@ float3 CustomPassLoadCameraColor(uint2 pixelCoords, float lod)
     }
 }
 
+
+float4 CustomPassSampleCustomColor(float2 uv)
+{
+    switch ((int)_CustomPassInjectionPoint)
+    {
+        default: return SampleCustomColor(uv);
+        case CUSTOMPASSINJECTIONPOINT_BEFORE_PRE_REFRACTION:
+        case CUSTOMPASSINJECTIONPOINT_BEFORE_POST_PROCESS:
+        case CUSTOMPASSINJECTIONPOINT_AFTER_POST_PROCESS: return SampleCustomColor(uv * _RTHandleScale.xy);
+    }
+}
+
+float4 CustomPassLoadCustomColor(uint2 pixelCoords)
+{
+    switch ((int)_CustomPassInjectionPoint)
+    {
+        default: return LoadCustomColor(pixelCoords);
+        case CUSTOMPASSINJECTIONPOINT_BEFORE_PRE_REFRACTION:
+        case CUSTOMPASSINJECTIONPOINT_BEFORE_POST_PROCESS:
+        case CUSTOMPASSINJECTIONPOINT_AFTER_POST_PROCESS: return LoadCustomColor(pixelCoords * _RTHandleScale.xy);
```

Comment:
Did you also test DLSS / TAAU up-scaling btw? I'm not sure, but perhaps when those are enabled you might also have to use the RTHandlePostProcessScale in some injection points?

Reply:
Good catch, it's indeed broken, I'll investigate the issue

Reply:
I pushed the fix